src/elements/element.line.js

```diff
@@ -184,4 +184,4 @@ class Line extends Element {
 
 Line.prototype._type = 'line';
 
-module.exports = Line;
+export default Line;
```

Comment:
Should we use the 
```js
export default class Line extends ...
```
style instead?

Thinking about style consistency when we get to helpers:
```js
export function helper1() {};
export function helper2() {};
```

vs
```js
function helper1() {}
function helper2() {}

export {
  helper1
  helper2
}
```

Reply:
I started playing with the helpers files. Personally I preferred the first style but I'm not opposed to the 2nd if that's what everyone else prefers

Reply:
I prefer brevity